src/Core/Ast/Parser/NikicPhpParser/FileReferenceVisitor.php

```diff
@@ -298,4 +309,18 @@ private function processClassLikeDocs(array $docNodeCrate): void
             }
         }
     }
+
+    /**
+     * @return array<string,string[]>
```

Comment:
`@return array<string, list<string>>` and then propagate everywhere.

Reply:
done